app/src/main/java/com/inveitix/android/clue/database/DBHelper.java

```diff
@@ -36,6 +41,10 @@ public void insertMuseum(Museum museum) {
         database.insert(DBConstants.DB_TABLE_MUSEUMS, null, cv);
     }
 
+    /**
+     * Inserts room data to rooms table
```

Comment:
Repeats what the name of the method says. Either elaborate or remove documentation.